<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorApp</class>
 <widget class="QWidget" name="CalculatorApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="resultContainer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>30</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#f8f8f8;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>100</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <item alignment="Qt::AlignRight|Qt::AlignBottom">
       <widget class="QLabel" name="resultLabel">
        <property name="styleSheet">
         <string notr="true">color: #404040;
font: 81 23pt &quot;Inter ExtraBold&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QLabel" name="operationLabel">
        <property name="styleSheet">
         <string notr="true">color: #404040;
font: 81 20pt &quot;Inter&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="calculatorContainer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>70</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ffffff;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="3">
       <widget class="QPushButton" name="buttonDivise">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background: #cd55e4;
	border: 0;
	margin: 10px;
	color: #fff;
	border-radius: 27%;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
	background: #bc4dd1;
}
</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonClear">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonPercent">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="buttonSeven">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonEight">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="buttonNine">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="buttonMultiply">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background: #f37476;
	border: 0;
	margin: 10px;
	color: #fff;
	border-radius: 27%;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
	background: #d16466;
}
</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="buttonFour">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="buttonFive">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="buttonSix">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="buttonThree">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="buttonPlus">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background: #fb8e19;
	border: 0;
	margin: 10px;
	color: #fff;
	border-radius: 27%;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
	background :#e37f14;
}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="buttonOne">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="buttonMinus">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background: #78b8f5;
	border: 0;
	margin: 10px;
	color: #fff;
	border-radius: 27%;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
	background :#689fd4;
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="buttonTwo">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonPlusMinus">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="buttonZero">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="buttonEqual">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background: #2dc982;
	border: 0;
	margin: 10px;
	color: #fff;
	border-radius: 27%;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
	background: #26ad70;
}</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="buttonDot">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0;
	color: #404040;
	font: 81 14pt &quot;Inter ExtraBold&quot;;
}

QPushButton:hover {
    background-color: #f8f8f8;
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
